### ISSUE TITLE
Bad dedent after return

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#299]
_[Trac time 20071101 195725Z]_ Reindenting the else line in this block of code:

```
        if stuff:
            return True
        else:
            return False
```

results in:

```
        if stuff:
            return True
     else:
            return False
```

**Comment by robmcmullen** [comment]
_[Trac time 20071104 234153Z]_ 8702a4cf9d9c81a2e7abfdd28d9a49b2831832e0 Fixed #299: used extra parsing info to determine if shouldn't dedent the else after a return

**Comment by robmcmullen** [comment]
_[Trac time 20071104 234155Z]_ 82047ee8e5afb084bc4747739b8ad5b35d536e90 Refs #299: fixed problem when indenting string -- wasn't returning extra requested data
